images/macos/provision/core/haskell.sh

```diff
@@ -14,6 +14,15 @@ for majorMinorVersion in $minorMajorVersions; do
     echo "install ghc version $fullVersion..."
     ghcup install $fullVersion
     ghcup set $fullVersion
+
+    # remove docs and profiling libs
+    ghc_bin_dir="$(ghcup whereis --directory ghc $fullVersion)"
+    [ -e "${ghc_bin_dir}" ] || exit 1
+    ghc_dir="$( cd "$( dirname "${ghc_bin_dir}" )" > /dev/null 2>&1 && pwd )"
```

Comment:
This works fine on my bash. But bash behavior can be different depending on version. I believe though this should work fine on mac too.